roles/confluent.kafka_connect/defaults/main.yml

```diff
@@ -29,11 +29,8 @@ kafka_connect:
   appender_log_file_size: 100MB
   properties:
     rest.port: "{{kafka_connect_rest_port}}"
-    consumer.interceptor.classes: io.confluent.monitoring.clients.interceptor.MonitoringConsumerInterceptor
-    producer.interceptor.classes: io.confluent.monitoring.clients.interceptor.MonitoringProducerInterceptor
-    config.storage.replication.factor: "{{ kafka_connect_default_internal_replication_factor }}"
-    config.storage.topic: "{{kafka_connect_group_id}}-configs"
-    group.id: "{{kafka_connect_group_id}}"
+    consumer.interceptor.classes: "{% if kafka_connect_metrics.enabled|bool %}io.confluent.monitoring.clients.interceptor.MonitoringConsumerInterceptor{% endif %}"
```

Comment:
How does this look in the .properties file when kafka_connect_metrics.enabled: false?

Reply:
As such:
`consumer.interceptor.classes=`